src/index.tsx

```diff
@@ -49,6 +49,7 @@ ReactDOM.render(
         client_id={AUTH_CONFIG.clientId}
         audience={AUTH_CONFIG.audience}
         redirect_uri={`${window.location.origin}/home`}
+        cacheLocation={AUTH_CONFIG.cacheLocation}
```

Comment:
You could try doing a webkit check here.

Reply:
Ok, thank you!